### ISSUE TITLE
ionic.bundle.js:26794 TypeError: Card is not a constructor

### ISSUE THREAD

**Issue by brunovinny** [gavruk/angular-card#23]
I tried to use this plugin on Ionic but I got this error message: "ionic.bundle.js:26794 TypeError: Card is not a constructor".
I'm using exactly the Example from this repository. What goes wrong?

**Comment by brunovinny** [comment]
Fixed.

**Comment by pradt** [comment]
Would be good if you could share what you did to fix this ?

**Comment by duranje** [comment]
Yes I'm having this issue as well... any advice here is greatly appreciated!

**Comment by RDMEC** [comment]
+1

**Comment by akafaneh** [comment]
+1

same issue here please advice !

**Comment by rgazeredo** [comment]
Look the folder example, documentation is not well explained in readme

**Comment by zurtri** [comment]
For me the solution was ensuring that the card.js file http://jessepollak.github.io/card/ was loaded before this directive.
